CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
- -
+ - Adds ability to sort order of courses in the same category.
+ - Adds locking of courses based on the order within their category.
+
+## [2.3.8]
+ - Adds a progress bar for courses with linear progress enabled.
```

Comment:
I'd agree on the format on writing changelog messages. We usually write about a change as it happened in the past, i.e. "Added a progress bar...". I'd consolidate the language we use across all changes.